cea/interfaces/arcgis/CityEnergyAnalyst.py

```diff
@@ -149,7 +149,7 @@ class LakePotentialTool(CeaTool):
     def __init__(self):
         self.cea_tool = 'lake-potential'
         self.label = 'Lake Potential'
-        self.description = 'Calculate the lake potential of the location'
+        self.description = 'Calculate the heat extracted from the Lake'
```

Comment:
is only heat extracted? or is the lake sometimes used as a heat sink? maybe the whole tool needs to be renamed?